builder/Builder.java

```diff
@@ -0,0 +1,382 @@
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.lang.module.FindException;
+import java.lang.module.ModuleDescriptor;
+import java.lang.module.ModuleFinder;
+import java.lang.module.ModuleReader;
+import java.lang.module.ModuleReference;
+import java.net.URI;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Supplier;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@SuppressWarnings("WeakerAccess")
+class Builder {
+
+  public static void main(String... args) throws Exception {
+    Builder builder = new Builder();
+
+    builder
+        .add(Line.forModule("org.joda.time").group("joda-time").artifact("joda-time"))
+        .project("Joda-Time")
+        .homepage("http://www.joda.org/joda-time");
+
+    builder
+        .add("org.joda", "joda-beans", "2.0")
+        .project("Joda-Beans")
+        .homepage("http://www.joda.org/joda-beans");
+
+    builder
+        .add("org.joda", "joda-beans")
+        .project("Joda-Beans")
+        .homepage("http://www.joda.org/joda-beans");
```

Comment:
Maybe the URL could be extracted from the pom.xml

Reply:
Good idea - as long there is a) an associated `pom.xml` and b) the URI is specified there.

Reply:
I'll add and use that pom-parsing feature when I implement a logical table section builder.

Something like this:

```java
var section = Section.of("org.joda").project("Joda").homepage("..."). // Section.of(<pom.xml>).
builder.add(section)
```